docs/quickstart/quickstart-non-docker.rst

```diff
@@ -0,0 +1,136 @@
+.. _ksql_quickstart_non_docker:
+
+Running KSQL Locally
+====================
+
+This topic describes how to setup a Kafka cluster and start KSQL in your local environment. After you complete these steps,
+you can return to the :ref:`create-a-stream-and-table` and start querying the data in the Kafka cluster.
```

Comment:
Perhaps add a link to the KSQL Docker instructions?  Think: "We also describe how to do this with Docker if you prefer that."  Just in case people don't follow the top-level navigation down to here, but find this page via Google Search.

Same backlink for the Docker instructions to this page (think: "If you don't want to use Docker, we also have ...")

Reply:
this doc appears in a table of contents and will show docker as well, a few inches to the left. i think it's best not to add extraneous (repeat) information. 

![image](https://user-images.githubusercontent.com/11722533/33102627-86e75c6e-ced2-11e7-9a0c-d1a411554044.png)